.github/workflows/diff-change-to-dist.yaml

```diff
@@ -51,9 +52,4 @@ jobs:
               diff +
               '\n```'
 
-            github.rest.issues.createComment({
-              issue_number: context.payload.pull_request.number,
-              owner: context.repo.owner,
-              repo: context.repo.repo,
-              body: commentText
-            })
+            await commentOnPr({github,context}, '## Changes to dist', commentText)
```

Comment:
Apologies if it's too early to comment or something you may have already tried. During the spike, [Ollie suggested using an HTML comment as a marker](https://github.com/alphagov/govuk-frontend/pull/3239#issuecomment-1422463273) rather than visible content from the comment. I think that's worth a shot as it'd allow to completely encapsulate what lets the comment be found again in `commentOnPr` and let us change the text at will 😊